File: individual-projects/2023/rudra-patel-200103102122/fundamental-practice/calculator.js
// calculator

function calculator() {
    let a = 10;
    let b = 20;
    let c = 30;
    let sum = given_sum(a, b, c);
    let subtraction = given_subtraction(a, b, c);
    let multiplication = given_multiplication(a, b, c);
    let division = given_division(a, b, c);
    console.log(
      "sum",
      sum,
      "subtraction",
      subtraction,
      "mul",
      multiplication,
      "div",
      division
    );
  }
  
  calculator();
  function given_sum(a, b, c) {
    return a + b + c;
  }
  function given_subtraction(a, b, c) {
    return a - c;
  }
  function given_multiplication(a, b, c) {
    return a * b;
  }
  function given_division(a, b, c) {
    return c / b;
  }

File: individual-projects/2023/ronak-bagdai-200103041003/Fundamental-Practice/Function/calculator.rs
fn add(a: i32,b: i32) -> i32 {
    return a + b;
}

fn sub(a: i32,b: i32) -> i32 {
    return a - b;
}

fn mul(a: i32,b: i32) -> i32 {
    return a * b;
}

fn div(a: i32,b: i32) -> i32 {
    return a / b;
}


fn main(){
    let sum = add(3,5);
    print!("The Sum is {}", sum);
    let sub = sub(7,5);
    print!("The Sub is {}", sub);
    let mul = mul(3,5);
    print!("The Mul is {}", mul);
    let div = div(3,5);
    print!("The Div is {}", div);
}